js/pages/vocabulary/components/search.html

```diff
@@ -79,7 +79,7 @@
           options: resultSources,
           optionsText: 'sourceName',
           optionsValue: 'sourceKey',
-          value: currentResultSource,
+          value: currentResultSource().sourceKey,
```

Comment:
This looks like 1currentResultSource()1 is an observable to a `Source` object, but should we be binding the dropdown to a full-source object instead of just a observable that stores what the current source selection?   I can't tell from this review, but what is in `currentResultSource()` and should we be updating the 1 field in it (sourceKey) when instead it would make more sense to just store the currentResultSource's key?  Maybe this var should be called `currentResultSourceKey`?